README.md

```diff
@@ -13,3 +31,12 @@ that ships with Vim 7.4, or a compatible file.
 For more information on the Hack language, visit http://hacklang.org.
 
 [pathogen]: https://github.com/tpope/vim-pathogen
+
```

Comment:
All stuff before this line should be above the list of link short names.  (Also, no reason to plug vim-plug in particular if you're not plugging Vundle, etc.; I'd just omit that.)